src/AssociationRegistry/Events/ContactgegevenVolgensKBOWerdGewijzigd.cs

```diff
@@ -0,0 +1,18 @@
+namespace AssociationRegistry.Events;
+
+using Framework;
+using System.Runtime.Serialization;
+using Vereniging;
+
+public record ContactgegevenVolgensKBOWerdGewijzigd(
+    int ContactgegevenId,
+    string Beschrijving,
+    bool IsPrimair) : IEvent
+{
+    [IgnoreDataMember]
+    public string Bron
```

Comment:
Since we're ignoring it, we can consider returning Bron as type instead of string